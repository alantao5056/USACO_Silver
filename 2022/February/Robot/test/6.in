40
-5 3
-3 0
5 4
-1 1
-4 -2
1 -1
-2 -1
-9 -7
-10 -7
-9 -6
-4 -4
-8 -5
5 5
-2 0
-3 0
3 2
-1 -3
-2 -1
-10 -8
4 1
3 1
-4 -3
0 3
-3 0
9 9
6 9
8 8
-10 -7
7 10
-1 -2
2 3
-2 -5
3 1
5 3
4 2
3 6
-3 0
9 6
8 9
7 8
9 8
