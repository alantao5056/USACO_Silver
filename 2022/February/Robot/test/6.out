0
0
14
132
1334
7735
37850
150977
509092
1485768
3762276
8413002
16603834
29194092
45780294
64246059
80784507
91047040
91974690
83178175
67165168
48322387
30788754
17304002
8489949
3601969
1304069
391711
96064
17782
2372
178
4
0
0
0
0
0
0
0
